File: Pattern20.go
    //         1
    //       2 1 2
    //     3 2 1 2 3
    //   4 3 2 1 2 3 4
    // 5 4 3 2 1 2 3 4 5

package main

import "fmt"

func main() {
	var n int
	fmt.Println("Enter the row number:")
	fmt.Scanf("%d",&n)

	for i:= 1; i <=n; i++ {
		for k:= i;k<=n;k++{
			fmt.Print("  ");
		}
		for j:= i;j>=1;j--{
			fmt.Print(j," ");
		}
		for p:= 2;p<=i;p++{
			fmt.Print(p," ");
		}
		fmt.Println()
	}
}

File: Pattern20.py
    #         1
    #       2 1 2
    #     3 2 1 2 3
    #   4 3 2 1 2 3 4
    # 5 4 3 2 1 2 3 4 5

n = int(input("Enter the number:"))
for i in range(1,n+1):
    for k in range(i,n+1):
        print("   ",end='')
    for j in range(i,0,-1):
        print(j," ",end='')
    for p in range(2,i+1):
        print(p," ",end='')
    print()
